doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -161,10 +189,12 @@ an opaque way. Nonetheless, the types described below are used to convey an spec
   greater than, lesser than, greater or equal, lesser or equal and range query operators. Eg:
 
 ```
+{
```

Comment:
I have added a textual clarification. As I said we need a correct JSON in order to allow Fabre to highlight properly. fixed https://github.com/jmcanterafonseca/fiware-orion/commit/01c2427080c7f44b53a89e26ce38de3faf33d990